samples/MvcSample.Web/Startup.cs

```diff
@@ -25,6 +25,65 @@ namespace MvcSample.Web
 {
     public class Startup
     {
+        public IServiceProvider ConfigureServices(IServiceCollection services)
+        {
+            services.AddCaching();
+            services.AddSession();
+
+            services.AddMvc();
+            services.AddSingleton<PassThroughAttribute>();
+            services.AddSingleton<UserNameService>();
+            services.AddTransient<ITestService, TestService>();
+
+            services.ConfigureMvc(options =>
+            {
+                options.Filters.Add(typeof(PassThroughAttribute), order: 17);
+                options.AddXmlDataContractSerializerFormatter();
+                options.Filters.Add(new FormatFilterAttribute());
+            });
+
+#if DNX451
+            //var applicationEnvironment = app.ApplicationServices.GetRequiredService<IApplicationEnvironment>();
+            //var configurationPath = Path.Combine(applicationEnvironment.ApplicationBasePath, "config.json");
```

Comment:
- this code was necessary to get the functional tests working reliably when they invoked this web site.  why is it no longer necessary here?
- if fully-qualifying the configuration path isn't needed here, why did you leave it in `Configure(IApplicationBuilder)`?
- do not check in commented-out code